csvcubed/csvcubed/cli/inspect/entrypoint.py

```diff
@@ -0,0 +1,32 @@
+"""
+CLI
+---
+
+The *Command Line Interface* for :csvcubed:`~csvcubed.csvcubed.cli.inspect`.
+"""
+
+from pathlib import Path
+import click
+from csvcubed.cli.inspect.inspect import inpsect
+
+from csvcubed.cli.inspect.metadatainputhandler import MetadataInputHandler
+
+
+@click.group(context_settings=dict(help_option_names=["-h", "--help"]))
+def entry_point() -> None:
+    """
+    csvcubed inspect - a CLI tool to validate CSV-W metadata files.
+
+    Member of :file:`./entrypoint.py`
+
+    :return: `None`
+    """
+
+
+@entry_point.command("inspect")
+@click.argument(
+    "csv", type=click.Path(exists=True, path_type=Path), metavar="TIDY_CSV_PATH"
```

Comment:
I think this will likely need to be the CSV-W's metadata file (i.e. a file with the extension `.csv-metadata.json`) instead of the CSV file itself.

Reply:
@robons You are correct - the extension will always be .csv-metadata.json in the use case for `csvcubed inspect`. I will update accordingly.